test/parallel/test-buffer-readint16be.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readInt16BE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readInt16BE(1);
+
+assert.strictEqual(result, 21672);
+
+assert.throws(
+  () => {
```

Comment:
Look at lib/net.js. The single statement es6 rockets have no braces, the multi-statement rockets have braces. Uses of single-statement with braces almost all have long statements which are being wrapped to next line, I guess because we don't like code like:

```
   someThing.shouldDoSomething(() =>
        andThenWeShould.doSomethingElse()
  );
```

But in above example, adding braces doesn't add any artificial new lines.

IMO, in the code being discussed here, the braces are syntactic noise that is ignored by javascript, but make it harder to read for people.

But if you all find this readable, its OK by me. But don't do it because its supposed to be like that, there are no rules about it that I can see.